sites/cheerpx/src/content/docs/20-faq.md

```diff
@@ -46,11 +46,33 @@ If you encounter the following error message:
 
 `Uncaught CheerpX initialization failed: DataCloneError: DedicatedWorkerGlobalScope.postMessage: The SharedArrayBuffer object cannot be serialized. The Cross-Origin-Opener-Policy and Cross-Origin-Embedder-Policy HTTP headers can be used to enable this.`
 
-This error occurs because CheerpX relies on SharedArrayBuffer, which requires the site to be cross-origin isolated. To activate cross-origin isolation, ensure your site is served over HTTPS and include the following headers in your responses:
+This error occurs because CheerpX relies on [SharedArrayBuffer], which requires the site to be [cross-origin isolated]. To activate cross-origin isolation, ensure your site is served over HTTPS and include the following headers in your responses:
 
 ```
 Cross-Origin-Embedder-Policy: require-corp
 Cross-Origin-Opener-Policy: same-origin
 ```
 
 By adding these headers to your server configuration you will enable cross-origin isolation and CheerpX will be able to start.
+
+For more information on server configurations, please see our [server setup guide].
+
+## Why Can't I Use the `file://` Protocol?
+
+CheerpX requires certain browser capabilities that are not available when accessing files directly using the `file://` protocol. Here’s an explanation of the issue and the recommended solution:
+
+### Security Restrictions and Cross-Origin Headers
+
+Modern browsers enforce strict security policies to protect users from potentially harmful actions. Accessing files through the `file://` protocol poses a security risk, as it can allow untrusted HTML files to interact with local resources. For example, opening random HTML files can access sensitive files on your disk, which may lead to data breaches.
```

Comment:
```suggestion
Modern browsers enforce strict security policies to protect users from potentially harmful actions. Accessing files through the `file://` protocol poses a security risk, as it can allow untrusted HTML files to interact with local resources. For example, opening random HTML files could access sensitive files on your disk, which may lead to data breaches.
```

Reply:
Also remove _all_ mentioned of the HTML-via-email case. It's just confusing.